### PR TITLE
python-packages: Add dynect python package

### PR DESCRIPTION
**Pull request by thedebugger**

### CODE REVIEW

**Comment by thedebugger** [comment]
Seems like a transist error. Closing the PR to retrigger the travis build.

**Comment by thedebugger** [comment]
Opening again.

**Comment by bjornfor** [comment]
Please don't mix whitespace cleanup and adding new packages.

File: lib/licenses.nix [hunk 1]
```diff
@@ -458,4 +458,8 @@ lib.mapAttrs (n: v: v // { shortName = n; }) rec {
     fullName = "Zope Public License 2.1";
   };
 
+  dynect = {
+    fullName = "Dyn Inc, Integration Team Deliverable";
```
Comment (FRidh) [inline lib/licenses.nix]: the licenses are ordered alphabetically. Could you move this to the correct position?

File: pkgs/top-level/python-packages.nix [hunk 2]
```diff
@@ -17265,4 +17265,30 @@ let
     };
   };
 
+  dyn = pkgs.pythonPackages.buildPythonPackage rec {
```
Comment (FRidh) [inline pkgs/top-level/python-packages.nix]: `pkgs.pythonPackages.` is not necessary. The order in this file is also mostly alphabetically. Could you put this in a better place?

**Comment by FRidh** [comment]
Aside from the other comments, could you rebase this? Looks good!

**Comment by domenkozar** [comment]
@thedebugger ping

**Pull request closed without merging**